<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KdReportWidget</class>
 <widget class="QWidget" name="KdReportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KD Report</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="editXml"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="toolsLayout">
         <item>
          <widget class="QToolButton" name="setKdHeadersButton">
           <property name="text">
            <string>Rename Columns</string>
           </property>
           <property name="icon">
            <iconset theme="insert-text">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="printKdReportButton">
           <property name="text">
            <string>Print</string>
           </property>
           <property name="icon">
            <iconset theme="printer">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="toolsSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="clearXmlButton">
           <property name="toolTip">
            <string>Clear Report description</string>
           </property>
           <property name="text">
            <string>Clear XML</string>
           </property>
           <property name="icon">
            <iconset theme="edit-clear">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="loadXmlButton">
           <property name="toolTip">
            <string>Load Report description</string>
           </property>
           <property name="text">
            <string>Load XML</string>
           </property>
           <property name="icon">
            <iconset theme="document-open">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="saveXmlButton">
           <property name="toolTip">
            <string>Save Report description</string>
           </property>
           <property name="text">
            <string>Save XML</string>
           </property>
           <property name="icon">
            <iconset theme="document-save">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="queryKdTable"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
